versions/3.0.md

```diff
@@ -1998,293 +1915,136 @@ paths:
-      operationId: getRepository
-      parameters:
-        username: $response.body#/owner/username
-        slug: $response.body#/slug
-    RepositoryPullRequests:
-      # returns '#/components/schemas/pullrequest'
-      operationId: getPullRequestsByRepository
-      parameters: 
-          username: $response.body#/owner/username
-          slug: $response.body#/slug
-    PullRequestMerge:
-      # executes /2.0/repositories/{username}/{slug}/pullrequests/{pid}/merge
-      operationId: mergePullRequest
-      parameters:
-        username: $response.body#/author/username
-        slug: $response.body#/repository/slug
-        pid: $response.body#/id
-  schemas: 
-    user: 
-      type: object
-      properties: 
-        username: 
-          type: string
-        uuid: 
-          type: string
-    repository: 
-      type: object
-      properties: 
-        slug: 
-          type: string
-        owner: 
-          $ref: '#/components/schemas/user'
-    pullrequest: 
-      type: object
-      properties: 
-        id: 
-          type: integer
-        title: 
-          type: string
-        repository: 
-          $ref: '#/components/schemas/repository'
-        author: 
-          $ref: '#/components/schemas/user'
-```
+##### OperationRef Examples
 
 As references to `operationId` MAY NOT be possible (the `operationId` is an optional 
 value), references MAY also be made through a relative `operationRef`:
 
 ```yaml
-components:
-  links:
-    UserRepositories:
-      # returns array of '#/components/schemas/repository'
-      operationRef: '#paths~12.0~1repositories~1{$response.body#/username}'
+links:
+  UserRepositories:
+    # returns array of '#/components/schemas/repository'
+    operationRef: '#paths~12.0~1repositories~1/{username}'
+    parameters:
+      username: $response.body#/username
 ```
 
 or an absolute `operationRef`:
 
 ```yaml
-components:
-  links:
-    UserRepositories:
-      # returns array of '#/components/schemas/repository'
-      href: 'https://na2.gigantic-server.com/#/paths/~12.0~1repositories~1{$response.body#/username}'
+links:
+  UserRepositories:
+    # returns array of '#/components/schemas/repository'
+    operationRef: 'https://na2.gigantic-server.com/#/paths/~12.0~1repositories~1{username}'
+    parameters:
+      username: $response.body#/username
 ```
 
 Note that in the use of `operationRef`, the _escaped forward-slash_ is necessary when 
 using JSON references.
 
+
+##### <a name="runtimeExpression"></a>Runtime Expressions
+
+Runtime expressions allow defining values based on information that will only be available within the HTTP message in an actual API call.
+This mechanism is used by [Link Objects](#linkObject) and [Callback Objects](#callbackObject).
+
+The runtime expression is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
+
+```
+      expression = ( "$url" | "$method" | "$request." [ source ] | "$response." [ source ])
```

Comment:
Good suggestion; I think just `$statusCode` is cleaner, since it does not complicate  `"$response." [ source ] `